node/blockAPI/shardBlock.go

```diff
@@ -111,3 +117,55 @@ func (sbp *shardAPIBlockProcessor) convertShardBlockBytesToAPIBlock(hash []byte,
 		Status:          BlockStatusOnChain,
 	}, nil
 }
+
+func (sbp *shardAPIBlockProcessor) setStatusESDTTransferTransactionsCrossShard(miniblocks []*api.MiniBlock) {
+	for _, mb := range miniblocks {
+		if mb.Type != block.TxBlock.String() {
+			continue
+		}
+
+		// ignore miniblocks that are not cross shard destination me
```

Comment:
instead of this comment, could have used a bool like:` isNotCrossShardMbDestinationMe`

Reply:
done